apps/api/src/schema/programs.ts

```diff
@@ -273,3 +280,48 @@ export const ugradRequirementsResponseSchema = z.object({
     .array(programRequirementSchema)
     .openapi({ description: "The requirements in this requirements block" }),
 });
+
+export const sampleProgramsYearSchema = z.object({
+  year: z.string().openapi({
+    description: "Year of this sample program",
+  }),
+  Fall: z.array(z.string()).openapi({
+    description: "Fall course list of this sample program",
+  }),
+  Winter: z.array(z.string()).openapi({
+    description: "Winter course list of this sample program",
+  }),
+  Spring: z.array(z.string()).openapi({
+    description: "Spring course list of this sample program",
+  }),
+});
+
+export const sampleProgramsResponseSchema = z.array(
+  z.object({
+    programName: z.string().openapi({
+      description: "Program name of this sample program",
+      example: "Biochemistry and Molecular Biology, B.S.",
+    }),
+    sampleProgram: z.array(sampleProgramsYearSchema).openapi({
+      description: "List of courses of this program",
+      example: [
+        {
+          year: "Freshman",
+          Fall: ["BIO SCI 93", "BIO SCI 93L", "CHEM 1A", "BIO SCI 2A"],
+          Winter: ["BIO SCI 94", "BIO SCI 94L", "CHEM 1B", "Lower-Division Writing1"],
+          Spring: [
+            "CHEM 1C- 1LC",
+            "STATS 7 or 8 (or Math 5A or General Education)",
+            "Lower-Division Writing1",
+          ],
+        },
+        {
+          year: "Sophomore",
+          Fall: ["BIO SCI 97", "CHEM 51A", "CHEM 1LD", "MATH 5A or 5B"],
+          Winter: ["BIO SCI 98", "CHEM 51B- 51LB", "MATH 5B (or General Education)"],
+          Spring: ["BIO SCI 99", "CHEM 51C- 51LC"],
+        },
```

Comment:
can you possibly include the notes here? see part below the table [here](https://catalogue.uci.edu/donaldbrenschoolofinformationandcomputersciences/softwareengineering_bs/#sampleprogramtext)